provider/offline.go

```diff
@@ -953,24 +964,44 @@ func (table *memoryOfflineTable) WriteBatch(recs []ResourceRecord) error {
 	return nil
 }
 
-type memoryMaterialization struct {
-	id   MaterializationID
-	data []ResourceRecord
+// Used in runner/copy_test.go
+type MemoryMaterialization struct {
+	// Lowercase do so it doesn't name thrash ID()
```

Comment:
??

Reply:
I'll make clearer, it can't be named Id because there is already an ID method on this struct